src/sentry/seer/similarity/similar_issues.py

```diff
@@ -200,19 +201,19 @@ def get_similarity_data_from_seer(
                 },
             )
         except SimilarGroupNotFoundError:
-            # This will similarly mark the entire request as errored even if it's only one group
-            # that we can't find, but again, we're okay with that because a group being missing
-            # implies there's something wrong with our deletion logic, which is a problem to which
-            # we want to pay attention.
-            #
-            # TODO: The deletion logic (tell Seer to delete hashes as soon as they're deleted on the
-            # Sentry side) comes with an inherent slight delay - the request to Seer happens async, and
-            # requests between services aren't instantaneous. It's therefore possible for us to land
-            # in a race condition, where Seer recommends a hash it doesn't know it's about to be
-            # asked to delete, and we come up empty because on the Sentry side the hash has already
-            # been deleted.
             parent_hash = raw_similar_issue_data.get("parent_hash")
 
+            # Tell Seer to delete the hash from its database, so it doesn't keep suggesting a group
+            # which doesn't exist
+            delete_seer_grouping_records_by_hash.apply_async(args=[project_id, [parent_hash]])
```

Comment:
Purely stylistic choice, feel free to ignore.

Personally I prefer `delay()` over `apply_async()` because it looks exactly like calling the original function and can be easily refactored to add or remove Celery by inserting or removing `.delay` between function name and its params. Also, because it preserves the signature, works better with MyPy.

Reply:
Oh, good call. I was pattern-matching off of our other calls to `delete_seer_grouping_records_by_hash`, but what you're saying makes a lot of sense. I'll change it.